index.html

```diff
@@ -5,17 +5,23 @@
     <title>Stream Service Viewer</title>
     <link
       rel="stylesheet"
-      href="https://js.arcgis.com/4.14/esri/themes/light/main.css"
+      href="https://js.arcgis.com/3.42/esri/css/esri.css"
     />
     <link
       rel="stylesheet"
-      href="https://js.arcgis.com/3.42/esri/css/esri.css"
+      type="text/css"
+      href="https://js.arcgis.com/calcite-components/1.0.0-beta.97/calcite.css"
```

Comment:
Is there a non-beta calcite release already?

Reply:
There is no non-beta release yet! The calcite-components repository is relatively new and there has been a big push lately to move toward it.